src/exampleTypes/Facet.js

```diff
@@ -123,8 +123,15 @@ let Facet = ({
     {_.flow(
       _.partition(x => _.includes(x.name, node.values)),
       _.flatten,
-      _.map(({ name, count }) => {
+      _.map(({ name, label, count }) => {
         let lens = tree.lens(node.path, 'values')
+        let invokeDisplayFn = (name, label) => {
+          if (_.isString(label)) {
+            return display(label)
```

Comment:
instead of doing this, we should just change the default display function to show label if it's a string. It would be weird for the signature of display to change based on whether or not there is a label